site/en/query/cquery.md

```diff
@@ -65,8 +65,8 @@ the [configuration](/reference/glossary#configuration) the
 target is built with.
 
 Since `cquery` runs over the configured target graph. it doesn't have insight
-into artifacts like build actions nor access to `[test_suite](/reference/be/general#test_suite)`
```

Comment:
If only we had static analysis checks to catch/auto-fix bugs like this!

Reply:
Heh, well, this one's pretty easily grep-able! That's how I found the other--just a quick search through this doc with the obvious regex.

In the meantime, though, your being great to work with makes it fun and rewarding to just patch 'em when I see 'em :)